skbeam/core/correlation.py

```diff
@@ -121,7 +119,7 @@ def _one_time_process(buf, G, past_intensity_norm, future_intensity_norm,
 
 results = namedtuple(
     'correlation_results',
-    ['g2', 'lag_steps', 'internal_state']
+    ['correlation_results', 'lag_steps', 'internal_state']
```

Comment:
Ah ok.  My primary complaint with calling this `correlation_results` is that it is just pretty long.  That being said, we should just call this whatever Andrei/Yugang want it to be called.  When you meet with them, can you also verify that the output is what they want to see?